internal/fuzzbinder/fuzzscraper.go

```diff
@@ -366,28 +381,48 @@ func GetFuzzTargets(fuzzParameters *FuzzParameters, date string) ([]string, erro
 
 // GetEvidences gets the list of the evidence files used by the fuzzscraper.
 // The expected date format is "YYYYMMDD".
-func GetEvidences(date string, fuzztargets []string, fuzzParameters *FuzzParameters) map[string]string {
-	evidences := make(map[string]string)
+func GetEvidences(date string, fuzzTargets []string, fuzzParameters *FuzzParameters) ([]amber.ClaimEvidence, error) {
+	evidences := make([]amber.ClaimEvidence, len(fuzzTargets)+2)
 	// Get the GCS absolute path of the file containing the revision hash of the source code used
 	// in the coverage build on a given day.
-	evidences["revision"] = fmt.Sprintf("gs://%s/%s/srcmap/%s.json", CoverageBucket, fuzzParameters.ProjectName, date)
+	role := "revision"
```

Comment:
Also, I am not sure what the role "revision" mean.

Reply:
> One important evidence that we want to have is the source code for the fuzz target. Right? We don't have that. Can you add a todo for adding that?
It is supposed to be easy to find using the commit hash since it has to be in the project itself, right?

Reply:
> Also, I am not sure what the role "revision" mean.

The evidence that helped to link the data to a given revision of the source code.

Reply:
> It is supposed to be easy to find using the commit hash since it has to be in the project itself, right? Do we still need to have it as evidence if it is on GitHub?

You are right. We have the exact path relative to the repo root in the perTarget reports, right?

Reply:
> The evidence that helped to link the data to a given revision of the source code. So, the evidence that helps get the commit hash.

Can you clearly document it somewhere? Also perhaps give it a better name :) What is the name of the file or bucket where this data came from?

Reply:
> You are right. We have the exact path relative to the repo root in the perTarget reports, right?

Yes, at least for oak. I'm not sure if the other projects using OSS-Fuzz have to put the fuzz-targets in `fuzz/fuzz_targets/`. I can add it as an input if that's not the case.

Reply:
How do get these paths? Cannot they be retrieved from the logs or other reports? I'd rather not add another input.

Reply:
> Can you clearly document it somewhere? Also perhaps give it a better name :) What is the name of the file or bucket where this data came from?

I will document this in the design doc. It is in `oss-fuzz-coverage` is a `srcmap` folder. I can name it `map date-revision`. What do you think?

Reply:
Somehow I feel like `srcmap` is better, but up to you. I think a clear documentation is more important.

Reply:
> How do get these paths? Cannot they be retrieved from the logs or other reports? I'd rather not add another input.

I'm not sure if they are available somewhere in the GCS bucket (I didn't see them). They are passed to OSS-Fuzz using `--target-dir`.

Reply:
> It is more readable to inline this.

I updated GetEvidences to make it more readable.

Reply:
> Somehow I feel like srcmap is better, but up to you. I think a clear documentation is more important.

I will use `srcmap` if it's more clear.

Reply:
The generated claim contains perTarget claims like the following:

```
                {
                    "name": "libFuzzer_oak_apply_policy",
                    "path": "oak/fuzz/fuzz_targets/apply_policy.rs",
                    "fuzzStats": {
                        "lineCoverage": "3.89% (215/5524)",
                        "branchCoverage": "0.00% (0/0)",
                        "detectedCrashes": false,
                        "fuzzTimeSeconds": 722232.9658203125,
                        "numberFuzzTests": 323023131
                    }
                },
```

I still don't understand where ` "path": "oak/fuzz/fuzz_targets/apply_policy.rs",` comes from. Is not it in some GCS file somewhere?

Reply:
In this version the path is fixed. It needs to be extracted from a coverage file in the future. I opened an issue and added it as a TODO.